packages/hoprd/src/index.ts

```diff
@@ -473,6 +480,30 @@ async function main() {
 
       logs.logStatus('READY')
       logs.log('Node has started!')
+
+      // Run a single command and then exit.
+      if (argv.run && argv.run !== '') {
```

Comment:
this code is no longer used, removed in https://github.com/hoprnet/hoprnet/commit/737b62316e758964a29dc3acf964a2c4ad3605d2

Reply:
Fixed in https://github.com/hoprnet/hoprnet/pull/4069/commits/80637b241dd811cb59d337ecffa6030e3776ccb2